maintainers/maintainer-list.nix

```diff
@@ -13894,4 +13894,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  tusj = {
```

Comment:
additions to maintainer list should be a separate commit
(ordered before the commit that introduces the use of the entry)